ProiectPractica/Validator/ConfigurationValidator.cs

```diff
@@ -0,0 +1,133 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Eventing.Reader;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProiectPractica.Validator
+{
+    public class ConfigurationValidator
+    {
+        /// <summary>
+        /// Validates a string so that it can be parsed into an integer.
+        /// Returns true if the string can be parsed as an integer,
+        /// and returns false otherwise.
+        /// </summary>
+        /// <param name="textToBeValidated"></param>
+        /// <param name="errorMessage"></param>
+        /// <returns></returns>
+        public bool ValidateInt(string textToBeValidated, out string errorMessage)
+        {
+            if (textToBeValidated.Length == 0)
+            {
+                errorMessage = "Field should not be empty!";
+                return false;
+            }
+            if (int.TryParse(textToBeValidated, out int integer))
+            {
+                if (integer < 0)
```

Comment:
You have a duplicated check for negative numbers, something is wrong, please re-write

Reply:
If the user tries to put -5.000.000.000, this cannot be parsed into an integer because it is smaller than min. value, so instead of saying that the number is too small, which can be a little bit confusing for the user, I said that the number is negative.